crhchttp/config.go

```diff
@@ -87,10 +86,11 @@ func (ac *AuthConfig) GetAccessToken(cxt context.Context, tokenURL string) error
 
 	body, err := io.ReadAll(resp.Body)
 
+	// TODO: handle errors for ServiceAccountData fields
 	// ONLY proceed to unmarshal if the status was 200
-	if resp.StatusCode != http.StatusOK {
-		return fmt.Errorf("status: %d | error response: %s", resp.StatusCode, body)
-	}
+	// if resp.StatusCode != http.StatusOK {
+	// 	return fmt.Errorf("status: %d | error response: %s", resp.StatusCode, body)
+	// }
```

Comment:
**WIP**: Handle errors specifically due invalid or missing (client-id, client-secret, grant-type) and provide an appropriate error message back to the user